pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1313,6 +1313,8 @@ type BRConfig struct {
 	SendCredToTikv *bool `json:"sendCredToTikv,omitempty"`
 	// OnLine specifies whether online during restore
 	OnLine *bool `json:"onLine,omitempty"`
+	// Image specifies the BR image used in the backup/restore
+	Image string `json:"image,omitempty"`
```

Comment:
@DanielZhangQD On a second thought, putting the `Image` under `BackupSpec` or `RestoreSpec` feels like it's the image for the tidb backup manager instead of the image for the underlying tool itself(BR/lightning/dumpling). I'm ok to put it there but do you think we should make the naming clearer?

Reply:
@namco1992 Your concern makes sense, what about `spec.toolImage`?